### PR TITLE
Patch/update checkout params

### PR DESCRIPTION
**Pull request by damilola-paystack**

### CODE REVIEW

File: example/src/main/java/com/paystack/checkout/example/MainActivity.kt [hunk 1]
```diff
@@ -50,12 +51,25 @@ class MainActivity : AppCompatActivity() {
...
+        println(metadata)
+
+        val checkoutBuilder = PaystackCheckout.Builder(this, "again@week.com",
+                50000, "NGN")
         checkoutBuilder.apply {
-            firstName("Michael")
-            lastName("Michael")
-            channels(PaymentChannel.bank, PaymentChannel.qr, PaymentChannel.card)
-            phone("07019133867")
+            reference("lezzGoLIvEagAIn41")
```
Comment (michael-paystack) [inline example/src/main/java/com/paystack/checkout/example/MainActivity.kt:71]: Hey @damilola-paystack, Is there a reason the calls for setting names and channels were taken out?

Comment (damilola-paystack) [reply example/src/main/java/com/paystack/checkout/example/MainActivity.kt:71]: @michael-paystack the first and last names are redundant. Metadata should be used instead. The public key picks the selected channel on the dashboard. Also, we should add the optional parameters in the ReadMe file.

Comment (damilola-paystack) [reply example/src/main/java/com/paystack/checkout/example/MainActivity.kt:71]: I will create another PR for the updated ReadMe

Comment (michael-paystack) [reply example/src/main/java/com/paystack/checkout/example/MainActivity.kt:71]: Okay. I'm thinking passing metadata isn't as type-safe as being able to set those values directly. I've always thought metadata would only be used where there are no other alternatives (e.g. custom fields).

Comment (damilola-paystack) [reply example/src/main/java/com/paystack/checkout/example/MainActivity.kt:71]: @michael-paystack while metadata might not be type-safe,  the lastName and firstName, if sent directly, will get dropped when it gets to our server. If the firstName and lastName are needed, the developer should call the create customer endpoint.

Comment (michael-paystack) [reply example/src/main/java/com/paystack/checkout/example/MainActivity.kt:71]: Okay. It's cool. I'll merge this now and we can talk about the other stuff later.

**Pull request merged**